### ISSUE TITLE
Relative urls in custom.scss have stopped working as of G5.3.3

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#1528]
@w00fz It appears that this commit https://github.com/gantry/gantry5/commit/18312b99d7ee86893f59a731862fd3beb5e57e33 may have broken existing sites that use relative urls in their custom.scss, Here are two reports from members in RT forum:

http://www.rockettheme.com/forum/joomla-template-sienna/262397-after-the-new-gantry-5-3-4-upgrade-the-image-urls-do-not-work

http://www.rockettheme.com/forum/gantry5-for-joomla/262407-relative-url-in-scss-files-problem-folder-structure-removed

Whilst I agree it's not ideal to be using such complex relative linking it does appear that the Gantry 5.3.3 update broke member sites that were working on G5.3.2

**Comment by mahagr** [comment]
Fixed in a3f443008fb894a3f6729f0a9e193d6eb0fe3c65

**Comment by marktaylor46** [comment]
For anyone who comes across this issue. It's is bad practice to use complex relative links to navigate directory structures - Best practice is to use the URI's that Gantry 5 provides - e.g.

Instead of writing:
```.mytest {
    background-image: url("../../images/backgrounds/base/img-01.jpg");
}```

Use this instead:
```.mytest {
   background-image: url("gantry-theme://images/backgrounds/base/img-01.jpg");
}```
